testing/test_assertion.py

```diff
@@ -354,6 +354,18 @@ def test_multiline_text_diff(self):
         assert "- spam" in diff
         assert "+ eggs" in diff
 
+    def test_bytes_diff(self):
+        diff = callequal(b"spam", b"eggs")
+        if PY3:
+            assert diff == [
+                "b'spam' == b'eggs'",
+                "At index 0 diff: 115 != 101",
```

Comment:
hmmm I thought that we *wouldn't* see this line, but only see the `- b'spam'` `+ b'eggs'` bit?

Reply:
No - it just skips the "contains more items" now.
I think it is fine / useful, although it could mention that those a bytes / ordinals.
If so, how should it sound like then?
Or should we skip any handling (i.e. as a sequence) for bytes?

Reply:
hmmm I'd like to still see the reprdiff since I think that's useful

Reply:
How about using the slice operator instead?

```
>>> b'foo'[0]
102
>>> b'foo'[0:1]
b'f'
```

We will then be able to see `"At index 0 diff: b's' != b'e'"`, which is readable and has no downsides as far as I can see.

Reply:
As a special case for bytes on py3 then, or for sequences in general?

Reply:
special case for bytes on py3, for sequences it is not clear to me it would be better than what we have now, at least for me.